idaes/power_generation/carbon_capture/piperazine_surrogates/co2_capture_system.py

```diff
@@ -0,0 +1,380 @@
+
+    def make_vars(self):
+        '''
+        This section builds port vars (Fc, T, P), CO2 capture rate
+
+        '''
+        # units declaration for vars
+        flow_units = pyunits.mol/pyunits.s
+        pressure_units = pyunits.Pa
+        temperature_units = pyunits.K
+        # heat_duty_units = pyunits.J/pyunits.s
+
+        # Component mole flows [mol/s]
+        self.inlet_flow_mol_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=1400/len(self.component_list),
+            units=flow_units,
+            doc='Inlet stream: Component mole flow [mol/s]')
+
+        self.pureCO2_flow_mol_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=1200/len(self.component_list),
+            units=flow_units,
+            doc='PureCO2 stream: Component mole flow [mol/s]')
+
+        self.exhaust_gas_flow_mol_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=100/len(self.component_list),
+            units=flow_units,
+            doc='exhaust_gas stream: Component mole flow [mol/s]')
+
+        # Temperature [K]
+        self.inlet_temperature = Var(self.flowsheet().config.time,
+                                     initialize=110,
+                                     units=temperature_units,
+                                     doc='Inlet temperature [K]')
+
+        self.pureCO2_temperature = Var(self.flowsheet().config.time,
+                                       initialize=110,
+                                       units=temperature_units,
+                                       doc='PureCO2 temperature [K]')
+
+        self.exhaust_gas_temperature = Var(self.flowsheet().config.time,
+                                           initialize=110,
+                                           units=temperature_units,
+                                           doc='exhaust_gas temperature [K]')
+
+        # Pressue [Pa]
+        self.inlet_pressure = Var(self.flowsheet().config.time,
+                                  initialize=17,
+                                  units=pressure_units,
+                                  doc='Inlet pressure [Pa]')
+
+        self.pureCO2_pressure = Var(self.flowsheet().config.time,
+                                    initialize=17,
+                                    units=pressure_units,
+                                    doc='PureCO2 pressure [Pa]')
+
+        self.exhaust_gas_pressure = Var(self.flowsheet().config.time,
+                                        initialize=17,
+                                        units=pressure_units,
+                                        doc='exhaust_gas pressure [Pa]')
+        # CO2 Capture rate
+        self.CO2_capture_rate = Var(self.flowsheet().config.time,
+                                    initialize=0.9,
+                                    doc='CO2 capture rate')
```

Comment:
If the surrogate is only valid for CO2_capture_rate = 0.9, can we add a note here?